Node.gitignore

```diff
@@ -3,6 +3,10 @@ logs
 *.log
 npm-debug.log*
 
+# .DS_Store
```

Comment:
We separate the OS-specific rules for a couple of reasons:
 
 - easier to maintain standalone OS files in one location than across many files
 - standalone OS files means consumers can tailor their setups more easily (e.g. add Windows and OSX rules to Node)

The template created initially is intended to help you get started - it should definitely be tweaked or extended based on your needs over time...